frontend/kick-covid-19/src/pages/Login.js

```diff
@@ -1,6 +1,8 @@
 import React from "react";
 import "./style.css";
 
+//import { useAuth } from '../context/auth.context';
+
 function Login() {
```

Comment:
and then you'll need something along these lines to actually wire up auth on the clientside with. your backend using the `useAuth` hook we built
```suggestion
function Login() {
  const. {login} = useAuth();

  handleLogin(evt) {
     // prevent default
     // get email and pass from inputs
     login(email, password);
     ...
 }
```